multi-libvterm.el

```diff
@@ -72,47 +72,44 @@ If this is nil, setup to environment variable of `SHELL'."
 ;;;###autoload
 (defun multi-libvterm (&optional _buffer-name)
   "Create new vterm buffer.
-Will prompt you shell name when you type `C-u' before this command."
+Will prompt you for the shell name when you type `C-u' before this command."
```

Comment:
I have checked again `C-u'. We don't need it. Can you pls help me review this docstring.